verifier/src/lib.rs

```diff
@@ -30,6 +30,13 @@ pub enum VerificationError {
         /// The actual attributes that were present
         actual: ConfigId,
     },
+    /// The config svn value of {actual} is less than the expected value of {expected}
+    ConfigSvnTooSmall {
```

Comment:
This name makes me wonder if the "mismatch" portion of the other errors should be renamed.

Reply:
I think the others are fine, since the comparison that would generate them is `==`, not `<=`/`>=`